data/policies/named_args.cas

```diff
@@ -0,0 +1,15 @@
+domain some_domain {
+	fn three_args(resource a, resource b, resource c) {
+		allow(this, a, file, write);
+		allow(this, b, file, read);
+		allow(this, c, file, open);
+	}
+
+	// TODO: It looks like using "this" as a class name is broken
```

Comment:
Not good :/

Reply:
Not great, no.  But it should be a pretty easy fix.  I'll have a PR for it later this month.

And that's just sort of incidental to this PR.  It came up because I wanted to use `this` in a class_name context as part of this PR.  So I think we can go ahead with this PR for now, and when I fix the bug mentioned here, I'll also convert this test to take advantage of it.